src/middlewared/middlewared/plugins/device_/device_info_linux.py

```diff
@@ -125,6 +125,10 @@ def get_rotational_rate(self, device_path):
         try:
             disk = libsgio.SCSIDevice(device_path)
             rotation_rate = disk.rotation_rate()
+        except RuntimeError:
```

Comment:
Can this exception only be raised for `qemu/kvm` guests or can it be raised otherwise as well ? Because in that case we would like to log the error ?

Reply:
Any system could fail here, but I chose not to log it since that's what we did previously. Do you feel it's necessary to log an error here considering it's expected on qemu/kvm systems?

Reply:
Nope, i am just seeing we are logging below but are choosing not to do so here. If you feel this is good, I am okay with that.

Reply:
I've added a little more logic. Let me know if this is better.